### DISCUSSION TITLE
OctaviOS sync failing issue

### DISCUSSION THREAD

**Discussion by GhostMaster69-dev** [ROM-builders/temporary#20140] [General]
After fixing sync issue by you there were two failed builds (but synced successfully):

lavander https://cirrus-ci.com/build/4554319148613632 with msm8998 hals

X01AD https://cirrus-ci.com/build/4650518992125952 with msm8996 hals

And somehow sync failed for me https://cirrus-ci.com/task/5832816290693120

**Comment by Apon77** [comment]
No idea why these happened.
Also failed these too.
hardware/qcom-caf/msm8996/display
hardware/google/camera
hardware/st/secure_element
packages/services/Car
